src/app/history/(components)/OrderTable/CancelOrdersDialog.tsx

```diff
@@ -41,7 +44,16 @@ export function CancelOrdersDialog({
         }) as OrderCancelArgs
     );
 
-    await sendTransactions(cancelTransactionsData);
+    await sendTransactions(cancelTransactionsData)
+      .then(({ safeTxHash }) => {
+        push(`/txpending/${safeTxHash}`);
+      })
+      .catch(() => {
+        toast({
+          title: "Error building transaction",
+          variant: "destructive",
+        });
+      });
```

Comment:
we standardized on using try/catch blocks, can you refactor this to use that pattern instead?